.github/workflows/build.yml

```diff
@@ -138,6 +139,14 @@ jobs:
         # note: system path must be modified in a previous step to it's use
         echo "$Env:VCPKG_INSTALLATION_ROOT\installed\${{ matrix.config.vc }}\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
         echo "${{github.workspace}}\build\openvdb\openvdb\${{ matrix.config.build }}" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+    - name: Setup NuGet Credentials
+      run: |
+        & $(vcpkg fetch nuget | select -last 1) sources add `
+        -source "https://nuget.pkg.github.com/${{ github.repository_owner }}/index.json" `
+        -storepasswordincleartext `
```

Comment:
what is `-storepasswordincleartext` doing here?

Reply:
I don't know I have just copied/pasted the recommended setup from the link mentioned above

Reply:
Thanks, will investigate but might take some time

Reply:
Not an expert on this, but after a quick look, I believe this flag only needs to be set on Linux/MacOS as dotnet only supports encryption on Windows.

Reply:
Removed!
Although the password stored is `"${{ secrets.GITHUB_TOKEN }}"`, nothing private is passed/stored.